docs/cpp_standards.md

```diff
@@ -5,10 +5,39 @@ title: Modern C++ standards
 Building Python wheels with modern C++ standards (C++11 and later) requires a few tricks.
 
 
-## manylinux1 and C++14
-The default `manylinux1` image (based on CentOS 5) contains a version of GCC and libstdc++ that only supports C++11 and earlier standards. There are however ways to compile wheels with the C++14 standard (and later): https://github.com/pypa/manylinux/issues/118
+## manylinux and C++
+The `manylinux*` standards imply a limit on which C++ standard you can use. There are workarounds to this, and `cibuildwheel` uses some workarounds by default. You should be aware of them!
 
-`manylinux2010` and `manylinux2014` are newer and support all C++ standards (up to C++17).
+Consider that the `manylinux*` standards constrain _symbol versions_ in `libstdc++.so`. So when *dynamically linking* to `libstdc++.so`, the desired `manylinux*` standard constrains the gcc version, and thus constrains the C++ standard.
+
+Cross-referencing [current manylinux standards](https://github.com/mayeut/pep600_compliance/blob/f86a7d7c153cc45aa3f2add6ffcf610c80501657/pep600_compliance/tools/policy.json) with [gcc's symbol versions](https://gcc.gnu.org/onlinedocs/libstdc++/manual/abi.html) and [libstdc++'s language support](https://gcc.gnu.org/onlinedocs/libstdc++/manual/status.html), we have:
+ * `manylinux1`: gcc 4.1.x
```

Comment:
This should probably be relegated to a note. For almost every user, this doesn't matter. IFAIK, all packages dynamically link and go about their day. This is a nice note, but what _most_ users what is the second table - what GCC version is _actually_ used. `manylinux1` is C++11 compatible, but not C++14 compatible, etc. This is good detail to have in a note just in case someone is debugging a really weird bug, or wondering how CentOS 5 has GCC 4.8.